File: app/src/main/java/org/metabrainz/mobile/data/sources/api/entities/acoustid/Date.java

package org.metabrainz.mobile.data.sources.api.entities.acoustid;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Date {

    @SerializedName("day")
    @Expose
    private int day;
    @SerializedName("month")
    @Expose
    private int month;
    @SerializedName("year")
    @Expose
    private int year;

    public int getDay() {
        return day;
    }

    public void setDay(int day) {
        this.day = day;
    }

    public int getMonth() {
        return month;
    }

    public void setMonth(int month) {
        this.month = month;
    }

    public int getYear() {
        return year;
    }

    public void setYear(int year) {
        this.year = year;
    }

}

File: app/src/main/java/org/metabrainz/mobile/data/sources/api/entities/acoustid/Date.kt
package org.metabrainz.mobile.data.sources.api.entities.acoustid

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Date {
    @SerializedName("day")
    @Expose
    var day = 0

    @SerializedName("month")
    @Expose
    var month = 0

    @SerializedName("year")
    @Expose
    var year = 0
}